content/interfaces-and-polymorphism/reading/introduction/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Introduction"
+date: 2023-02-01T13:10:40-06:00
+draft: false
+weight: 1
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The final pillar of object-oriented programming that we’ll
+explore is polymorphism.
+
+## Polymorphism
+
+**Polymorphism** is an object-oriented mechanism that allows for objects
+of different types to be used in the same way.
+
+<!-- TODO: Add link back to inheritance -->
+
+We’ve already encountered polymorphism made possible by inheritance when talking about 
+casting with our `Cat` and `HouseCat` classes.
+In that case, we stored an object of type `HouseCat` in its compatible type, `Cat`.
+
+Let's take a closer look at how polymorphism might work in our cat-centric application.
+
```

Comment:
Is there example code for this chapter in a repo?

Reply:
There is no example code for this chapter.